server/index.ts

```diff
@@ -105,12 +106,16 @@ import {Ladders} from './ladders';
 global.Ladders = Ladders;
 
 import {Chat} from './chat';
+// tslint:disable-next-line: no-floating-promises - cannot be propogated any higher
+Chat.loadTranslations();
```

Comment:
Remove tslint comment and 
```suggestion
void Chat.loadTranslations();
```

Reply:
I'm not sure about this. I like the clarity / explicitness of the TSLint comment. Maybe voiding it + comment is a good compromise?

Reply:
I'm not sure what the comment adds that the `void` doesn't?

(Also it's spelled "propagated".)

I think if it only happened once ever, it would make sense, but if you want to comment `void` every time, I mean, just explain what `void` means in `CONTRIBUTING.md`. The purpose of a comment is definitely _not_ to explain what a keyword means.

Reply:
> I think if it only happened once ever, it would make sense, but if you want to comment void every time, I mean, just explain what void means in CONTRIBUTING.md. The purpose of a comment is definitely not to explain what a keyword means.

There are other cases that are not just "cannot be propagated higher" where I'd like to keep a comment, like https://github.com/TheJetOU/Pokemon-Showdown/blob/propogate-promises/server/room-battle.ts#L561.

So, I'll just use ``void`` with no comment if it simply cannot be propagated higher, and ``void`` + comment in those special cases.